README.md

```diff
@@ -63,6 +63,13 @@ control "aws-1" do
 end
 ```
 
+### Vendor your Inherited Profile
+
+Your `depends:` above makes your `my-profile` an inherited InSpec profile. (see:[Profile Dependencies](https://www.inspec.io/docs/reference/profiles/index.html#profile-dependencies) in the InSpec docs) Prior to using your `my-profile` make sure you run:
```

Comment:
This is not correct. `my-profile` is the including profile, `aws` is the inherited profile. But again, I think this is out-of-scope for this README and a link to the existing docs is all that's necessary. This is a short-term need anyways.